packages/web-platform/web-tests/package.json

```diff
@@ -14,7 +14,7 @@
     "build:cases": "rm -rf dist && node  ./scripts/generate-build-command.js",
     "coverage": "nyc report --cwd=$(realpath ../)",
     "coverage:ci": "nyc report --cwd=$(realpath ../) --reporter=lcov",
-    "lh": "pnpm dlx @lhci/cli autorun",
+    "lh": "node ./scripts/run-lighthouse.js && pnpm dlx @lhci/cli upload",
```

Comment:
Could we turn this script into a standalone package that can be executed with `pnpm dlx`? This would allow us to avoid installing the `lighthouse` package in our workspace, saving significant time on CI.